src-tauri/src/main.rs

```diff
@@ -17,6 +19,16 @@ fn main() {
         .on_system_tray_event(tray::system_tray_handler);
 
     builder
+        .plugin(tauri_plugin_single_instance::init(|app, _, _| {
+            let window = app.get_window("main").unwrap();
+            let _ = window.set_focus();
+            if !window.is_visible().unwrap() {
+                let _ = window.show();
+            }
+            if window.is_minimized().unwrap() {
+                let _ = window.unminimize();
+            }
```

Comment:
Wouldn't this result in the tray toggle menu item being out of sync? I think, it would make sense to make use of https://github.com/cinnyapp/cinny-desktop/blob/6f83708511f923d6561bd5f5357049a698361a43/src-tauri/src/tray.rs#L43 here.

Reply:
you are right. Thanks for the hint.
I updated the code, but I can't use the toggle method, as we don't want to hide the window if already visible